alerts/cluster-monitoring-operator/ThanosRuleRuleEvaluationLatencyHigh.md

```diff
@@ -1,4 +1,4 @@
-# ThanosRuleEvaluationLatencyHigh
+# ThanosRuleRuleEvaluationLatencyHigh
```

Comment:
Is `RuleRule` intentional?

Reply:
yes https://github.com/openshift/cluster-monitoring-operator/blob/6fbfcbce3c5ff8a58ce3141e9d5a8fa0c7d20dae/assets/thanos-ruler/thanos-ruler-prometheus-rule.yaml#L56

I see thats how all thanosrule alerts are named (prefixed with `ThanosRule`)